tools/fmt/fmt.rs

```diff
@@ -208,12 +222,17 @@ fn format_element(
     state: &mut FormatState,
 ) -> Result<(), std::io::Error> {
     let mut sub = node.children_with_tokens();
-    if !(whitespace_to(&mut sub, SyntaxKind::QualifiedName, writer, state, "")?
-        && whitespace_to(&mut sub, SyntaxKind::LBrace, writer, state, " ")?)
```

Comment:
Same here and everywhere, the `&&` was short-circuiting the call to whitespace_to.